functions/src/auth.js

```diff
@@ -1,3 +1,5 @@
+// TODO: Rename this file to auth.ts (or auth/index.ts) + refactor to make proper use of TypeScript
```

Comment:
[TODO] Rename this file to `auth.ts` (or `auth/index.ts`) + refactor to make proper use of TypeScript